crates/sui-framework/packages/sui-system/sources/staking_pool.move

```diff
@@ -305,6 +305,14 @@ module sui_system::staking_pool {
         option::is_some(&pool.deactivation_epoch)
     }
 
+    /// Returns a boolean value indicating if the StakingPool is in-active or not, the epoch number the StakingPool was de-activated at if it is inactive.
+    public (friend) fun get_deactivation_epoch(pool: &StakingPool): (bool, u64) {
```

Comment:
Why not just return an `Option<u64>`? Same question for the other getters.

Reply:
no particular reason, i handled the case when `deactivation_epoch` is not defined by returning 0 so the module calling this function doesn't have to handle `Option` and can directly work with returned `deactivation_epoch` value. 

I can return `Option<u64>` if that's preferred. Just lmk what's the preferred response here.